lib/Listener/ExifMetadataProvider.php

```diff
@@ -95,9 +95,9 @@ public function handle(Event $event): void {
 				$gps['longitude'] = $this->gpsDegreesToDecimal($rawExifData['GPS']['GPSLongitude'], $rawExifData['GPS']['GPSLongitudeRef']);
 			}
 
-			if (array_key_exists('GPSAltitude', $rawExifData['GPS']) && array_key_exists('GPSAltitudeRef', $rawExifData['GPS'])) {
-				$gps['altitude'] = ($rawExifData['GPS']['GPSAltitudeRef'] === "\u{0000}" ? 1 : -1) * $this->parseGPSData($rawExifData['GPS']['GPSAltitude']);
-			}
+			if (array_key_exists('GPSAltitude', $rawExifData['GPS']) && array_key_exists('GPSAltitudeRef', $rawExifData['GPS']) && is_string($rawExifData['GPS']['GPSAltitude'])) {
+				$gps['altitude'] = ($rawExifData['GPS']['GPSAltitudeRef'] === '1' ? -1 : 1) * $this->parseGPSData($rawExifData['GPS']['GPSAltitude']);
```

Comment:
Is this change necessary ?

Reply:
This was originally `$rawExifData['GPS']['GPSAltitudeRef'] === "\u{0000}" ? 1 : -1)` \u{0000} is supposed to be the null character but the specifications say it can also be 0 for above sea level. A value of 1 indicates below sea level.

Reply:
Hmm, alright then. Have you tested it?

Reply:
Yes. I tested with this code and expected values were logged.


```
			if (array_key_exists('GPSAltitude', $rawExifData['GPS']) && array_key_exists('GPSAltitudeRef', $rawExifData['GPS']) && is_string($rawExifData['GPS']['GPSAltitude'])) {
				$this->logger->info('GPSAltitude: ' . $rawExifData['GPS']['GPSAltitude']);
				$this->logger->info('GPSAltitudeRef: ' . $rawExifData['GPS']['GPSAltitudeRef']);
				$gps['altitude'] = ($rawExifData['GPS']['GPSAltitudeRef'] === '1' ? -1 : 1) * $this->parseGPSData($rawExifData['GPS']['GPSAltitude']);
				$this->logger->info('GPSAltitudeCalc: ' . $gps['altitude']);
			}
```

Reply:
I tested with a photo that I used exiftool to write metadata for below sea level and it didn't work.  The logged value for GPSAltitudeRef was "\u0001" so I will revert it to using the unicode character but assuming above sea level if the value isn't explicitly below.